.github/workflows/conformance-aks.yaml

```diff
@@ -250,18 +251,11 @@ jobs:
             --helm-set=debug.enabled=true \
             --helm-set loadBalancer.l7.backend=envoy \
             --helm-set tls.secretsBackend=k8s \
-            --azure-resource-group ${{ env.name }}-${{ matrix.location }} \
-            --wait=false \
-            --rollback=false \
```

Comment:
What happened to wait, rollback?

Reply:
- in helm mode we changed the default to false for `--wait` flag: https://github.com/cilium/cilium-cli/blob/main/internal/cli/cmd/install.go#L252
- we didn't implement `--rollback` flag in helm mode because it doesn't leave dangling resources when the installation fails. we needed this flag in the classic mode in case a resource creation failed midway during the installation to clean up resources that already got created.